### PR TITLE
Update open telemetry

### PR DESCRIPTION
**Pull request by Johnabell**
Currently this is blocked because a change in Open telemetry requires the protoc compile to be included in the build.

We could add this to the base container. However, it looks like they have backed out this change but it will not land until version `0.19.0` is released

https://github.com/open-telemetry/opentelemetry-rust/pull/881
https://github.com/open-telemetry/opentelemetry-rust/issues/873

### CODE REVIEW

**Review by cpiemontese — APPROVED** [review]
Thanks :pray:

**Review by MaeIsBad — APPROVED** [review]

**Comment by Johnabell** [comment]
I don't have merge permissions, so could someone please merge this for me when appropriate.

**Pull request merged**
